tempesta_fw/connection.h

```diff
@@ -146,12 +146,21 @@ typedef struct {
  * @seq_queue	- queue of client's messages in the order they came;
  * @seq_qlock	- lock for accessing @seq_queue;
  * @ret_qlock	- lock for serializing sets of responses;
+ * @msg_stream	- currently streamed response to client;
+ * @conn_stream	- server connection where current request is streamed to;
+ * @recv_window_sz - maximum size of memory used to store requests, used for
+ *		     receive window size steering;
+ * @buff_sz	- current size of memory used to store requests;
  */
 typedef struct {
 	TFW_CONN_COMMON;
 	struct list_head	seq_queue;
 	spinlock_t		seq_qlock;
 	spinlock_t		ret_qlock;
+	TfwMsg			*msg_stream;
+	TfwConn			*conn_stream;
+	size_t			recv_window_sz;
```

Comment:
The variable isn't used, it seems you just forgot to do the most important change for https://github.com/tempesta-tech/tempesta/pull/1012#discussion_r187219535